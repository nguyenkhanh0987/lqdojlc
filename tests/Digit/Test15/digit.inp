100
234034890 278464588
83333669 642558325
336629586 430076265
661680391 860220820
322337686 326134932
595873872 730637316
791721609 819344723
398124054 722465082
115757781 759750902
236585112 842694180
47524459 623237742
198227689 420037667
897027555 973709105
310074376 554522576
983565613 993217591
823962451 891664222
599640261 740575531
751125314 844601009
428638610 875528669
242281539 258262689
135017426 618213807
485277345 779017398
428803699 495744684
50206602 836299508
826182168 945658286
810121232 968521428
182906859 963440516
576150404 985687883
139704687 333707502
783466650 923059567
914994113 994958579
638380417 776093311
959080280 964264996
471944021 571704862
492535860 505950895
621999911 700279530
914899661 921149299
553895130 802603843
160420739 280572265
807419399 919948065
19962880 865921801
294094848 349599682
106534729 931372172
165883994 345138873
188460156 936843279
428741301 815588692
617292413 942189815
968275788 969400644
814192045 838320073
260895375 682259233
233301217 780750398
642675709 950529102
281548241 828869440
922029479 956045483
735648125 832361691
95019470 771931745
383887037 959848156
361021350 426513714
537003976 936359297
3459668 48376359
36426556 456128116
124996436 239358329
316386300 821003343
26609075 152336510
172091072 281356181
929295912 936549324
658421854 881381343
31952370 879170307
18354782 77108438
156456407 689965271
661580702 774750424
988191593 994895260
766892232 815209760
179889597 433997837
800680242 862448959
469414006 959597427
113266481 347012809
957397045 992001287
964153800 969997365
976194530 987328276
146508286 460497610
300334791 378005514
631040450 635935319
861290999 877418382
64145727 424845704
871139783 912259370
322859586 518314722
653188298 836062098
734303139 839957635
895764326 962502713
456673628 491375445
528594578 596473329
848741019 984295278
708842242 743452668
122573892 999609659
485527484 881531055
991550061 995064301
556085516 951645724
365405412 796820715
969013072 988597119
