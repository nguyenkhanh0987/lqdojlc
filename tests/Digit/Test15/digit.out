8079926
84587221
13996715
17138172
449078
11988812
1713665
36871427
89006247
74451210
91880724
34472339
3554796
32832506
1084140
5426236
12051018
7794515
46529913
2731692
73491874
30590651
10742061
110473877
7850259
9831220
91328426
34881176
34553211
10157254
5273132
13434290
494056
12358222
1907942
9129832
115137
26414680
22369649
7430143
118104699
6610021
103482438
30646474
88445053
42650567
26559160
25857
900869
57244112
70398525
25128754
65217060
1503419
8142752
93708644
57265159
10281426
36410430
7301293
72938861
20486930
58940653
24913432
19849147
332725
19710577
117149306
12284584
77143592
10999123
912689
4885047
41000955
3002216
46233350
39962781
3125744
251122
1079495
51248014
10533719
446248
1610343
63377413
3762696
28011136
16579625
8478886
2784844
6156894
9604381
9707527
1851358
106722983
38535042
564324
34983437
51333469
1882762
